### ISSUE TITLE
SyntaxError: Unexpected token c

### ISSUE THREAD

**Issue by Arcanemagus** [AtomLinter/linter-csslint#93]
When calling `lint()` on an empty file the following error is thrown:

```text
SyntaxError: Unexpected token c
    at Object.parse (native)
    at file:///C:/Users/Landon Abney/.atom/packages/linter-csslint/lib/main.coffee:26:29
    at process._tickCallback (node.js:368:9)
```

**Comment by Arcanemagus** [comment]
Interestingly, this doesn't trigger on my normal profile, only on a blank one.
